suite-common/wallet-utils/src/exportTransactionsUtils.ts

```diff
@@ -157,6 +166,13 @@ const prepareContent = (data: Data, tokenDefinitions: TokenDefinitions): Fields[
                 type: t.type.toUpperCase(),
                 txid: t.txid,
             };
+            const fiatRateKey = getFiatRateKey(
+                t.symbol,
+                data.localCurrency as FiatCurrencyCode,
```

Comment:
I think the cast here is not needed, the `string` type in `Data` can be changed to `FiatCurrencyCode` and it should most probably work